### PR TITLE
Update build badge on Readme

### PR DESCRIPTION
**Pull request by Arhell**
resolve https://github.com/dwyl/hits/issues/141

### CODE REVIEW

**Comment by Arhell** [comment]
cc @nelsonic

**Comment by Arhell** [comment]
cc @SimonLab

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 <div align="center">
 
-[![Build Status](https://img.shields.io/travis/dwyl/hits/master.svg?style=flat-square)](https://travis-ci.org/dwyl/hits)
+![Build Status](https://github.com/dwyl/hits/actions/workflows/ci.yml/badge.svg)
```
Comment (nelsonic) [inline README.md:7]:
```suggestion
[![GitHub Workflow Status](https://img.shields.io/github/workflow/status/dwyl/hits/Elixir%20CI?label=build&style=flat-square)](https://github.com/dwyl/hits/actions/workflows/ci.yml)
```

The standard badge is: Build Status

We prefer the "Flat Square": GitHub Workflow Status

Also hyperlinked to the build history: https://github.com/dwyl/hits/actions/workflows/ci.yml
image

**Review by nelsonic — APPROVED** [review]
@Arhell thanks very much for making this update to fix #141 ✅

**Pull request merged**